bin/clear.py

```diff
@@ -1,2 +1,4 @@
 """Clear the stash console output window"""
-print u'\u009bc',
+# TODO: prevent linebreak after 0099bc
+# This was removed as a quick fix
+print(u'\u009bc')
```

Comment:
`print(..., end="")` (requires `from __future__ import print_function`)

Reply:
Huh, didnt know that you could use `__future__` imports from withing StaSh. Seems like there is still so much to learn :)